modules/BundledPackages/BundledPackages.proj

```diff
@@ -13,7 +14,30 @@
       BuildInParallel="true"
       RemoveProperties="PublishDir" />
 
-    <Copy SourceFiles="module.props" DestinationFolder="$(PublishDir)" />
+    <PropertyGroup>
+      <!--
+        This is a workaround for https://github.com/Microsoft/msbuild/issues/2914.
+        Currently, the only way to configure the NuGetSdkResolver is with NuGet.config, which is not generally used in aspnet org projects.
+        This project is restored so that it pre-populates the NuGet cache with SDK packages.
+      -->
+      <BundledPackageRestorerContent>
+<![CDATA[
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFramework>netcoreapp2.0</TargetFramework>
+    <DisableImplicitFrameworkReferences>true</DisableImplicitFrameworkReferences>
```

Comment:
Does this need `RestoreSources` or does it pick the one via Directory.Build.props?

Reply:
It's set in the PreinstallBundledPackages target (see below). We can't set it here since the path will be machine-specific.